### PR TITLE
Return the tags of the task in the json callback results

### PR DESCRIPTION
**Pull request by ctria**
This returns the tags of the task in the json results. This is additional information to be returned and useful if you need to categorize your results based on the tags of the tasks.

### CODE REVIEW

**Comment by maxamillion** [comment]
cc @bcoca @sivel

File: lib/ansible/plugins/callback/json.py [hunk 1]
```diff
@@ -109,6 +109,7 @@ def _record_task_result(self, on_info, result, **kwargs):
         task_result = result._result.copy()
         task_result.update(on_info)
         task_result['action'] = task.action
+        task_result['tags'] = task.tags
```
Comment (sivel) [inline lib/ansible/plugins/callback/json.py:112]: I don't think tags should go into the result.

Maybe instead:

```
diff --git a/lib/ansible/plugins/callback/json.py b/lib/ansible/plugins/callback/json.py
index c04dd49e23..f6b932e4be 100644
--- a/lib/ansible/plugins/callback/json.py
+++ b/lib/ansible/plugins/callback/json.py
@@ -60,7 +60,8 @@ class CallbackModule(CallbackBase):
         return {
             'task': {
                 'name': task.get_name(),
-                'id': str(task._uuid)
+                'id': str(task._uuid),
+                'tags': task.tags,
             },
             'hosts': {}
```

**Review by sivel — CHANGES_REQUESTED** [review]

File: lib/ansible/plugins/callback/json.py (hunk 1 above)
Comment (bcoca) [reply lib/ansible/plugins/callback/json.py:112]: agreed, its not part of the result, it is part of teh task

**Comment by sivel** [comment]
Thank you very much for this submission.  Changes were requested on this PR, but no changes have been made. Due to the age of this PR I am going to proceed with closing it. Should you wish to make the requested updates, please open a new PR.

**Pull request closed without merging**